scripts/lib/common.sh

```diff
@@ -28,3 +28,68 @@ function display_timelines() {
     echo "TIMELINE: Conformance tests took $CONFORMANCE_DURATION seconds."
     echo "TIMELINE: Down processes took $DOWN_DURATION seconds."
 }
+
+function run_warm_ip_test() {
```

Comment:
little nit, but the `lib/common.sh` file was really for general-purpose Bash functions, not necessarily for functions that run a specific test, etc. Consider placing this and the below function into a separate `lib/warm_tests.sh` file? Also, consider putting a comment above each function explaining roughly what the purpose of the test is... In this case, the test is merely checking the propogation of environment variables in Kubernetes Daemonset -> Pod, but isn't actually testing the **functionality** of the warm IP pool.